### PR TITLE
Fixing deployment to git master

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

**Comment by mermop** [comment]
This could probably do with some documentation about how it works - there's a few magical branch names etc that might be worth explaining for future developers

**Comment by Br3nda** [comment]
@mermop I've just pushed a note in `README.md` about the branches

File: README.md [hunk 1]
```diff
@@ -135,3 +135,13 @@ We must meet the [New Zealand Government - Web Accessibility Standard 1.1](https
...
+
+## Git Branches
+
+Jekyll config (including markdown, and layout html) are in the `dev` branch, which is also the default branch on github.
+
+The static site is built by circle-ci. The deploy task runs on any change to the dev branch. (see `.circlci/config.yml`).
+
+The status content (html, css, js, images) is then commited to the master branch. Github Pages is configured for this repo to serve from the master branch.
+
+Note: Github Pages for organisaions must be served from the `master` branch. At this time the branch name if not configurable.
```
Comment (mermop) [inline README.md]:
```suggestion
Note: Github Pages for organisations must be served from the `master` branch. At this time the branch name is not configurable.
```

File: README.md [hunk 2]
```diff
@@ -135,3 +135,13 @@ We must meet the [New Zealand Government - Web Accessibility Standard 1.1](https
...
 
 The tests will not catch every accessibility problem.  We still should spend time checking the toolkit, and generally keeping proficient accessibility standards.
+
+## Git Branches
+
+Jekyll config (including markdown, and layout html) are in the `dev` branch, which is also the default branch on github.
+
+The static site is built by circle-ci. The deploy task runs on any change to the dev branch. (see `.circlci/config.yml`).
+
+The status content (html, css, js, images) is then commited to the master branch. Github Pages is configured for this repo to serve from the master branch.
```
Comment (mermop) [inline README.md]:
```suggestion
The status content (html, css, js, images) is then committed to the master branch. Github Pages is configured for this repo to serve from the master branch.
```

File: README.md [hunk 3]
```diff
@@ -135,3 +135,13 @@ We must meet the [New Zealand Government - Web Accessibility Standard 1.1](https
...
 
 The tests will not catch every accessibility problem.  We still should spend time checking the toolkit, and generally keeping proficient accessibility standards.
+
+## Git Branches
+
+Jekyll config (including markdown, and layout html) are in the `dev` branch, which is also the default branch on github.
+
+The static site is built by circle-ci. The deploy task runs on any change to the dev branch. (see `.circlci/config.yml`).
+
+The status content (html, css, js, images) is then committed to the master branch. Github Pages is configured for this repo to serve from the master branch.
```
Comment (mermop) [inline README.md]: do you mean 'static content' rather than 'status content' here?

Comment (Br3nda) [reply README.md]: yes

**Pull request merged**